pynamodb/models.py

```diff
@@ -888,7 +899,7 @@ def _get_schema(cls) -> ModelSchema:
 
         return schema
 
-    def _get_save_args(self, condition: Optional[Condition] = None, *, add_version_condition: bool = True) -> Tuple[Iterable[Any], Dict[str, Any]]:
+    def _get_save_args(self, condition: Optional[Condition] = None, *, add_version_condition: bool = True, return_values_on_condition_failure: Optional[str] = None) -> Tuple[Iterable[Any], Dict[str, Any]]:
```

Comment:
This could be typed as:

```suggestion
    def _get_save_args(self, condition: Optional[Condition] = None, *, add_version_condition: bool = True, return_values_on_condition_failure: Optional[Literal["ALL_OLD", "NONE"]] = None) -> Tuple[Iterable[Any], Dict[str, Any]]:
```